tools/QuietBackgroundProcesses/DevHome.QuietBackgroundProcesses.Common/empty.cpp

```diff
@@ -0,0 +1,2 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
>Copyright [](http://example.com/codeflow?start=3&length=9)

Why is this file?

Reply:
// A cpp is required to produce a .lib, and a .lib is required to exist if this project is
// referenced by another cpp project.

(afaict *shrugs*)

Reply:
Not necessarily. [Here](https://github.com/microsoft/devhome/tree/microsoftfeature/dev-environments/HyperVExtension/src/DevSetupEngineIdl) is a vcproj that I created with only one idl file which it compiles to produce .winmd.

Reply:
I took a look - but I don't see the same setup - a vcxproj project that consumes the vcxproj idl project as a reference (which then would require the .lib).